icons/plug.svg

```diff
@@ -10,7 +10,7 @@
   stroke-linejoin="round"
 >
   <path d="M12 22v-5" />
-  <path d="M9 7V2" />
-  <path d="M15 7V2" />
+  <path d="M9 8V2" />
+  <path d="M15 8V2" />
   <path d="M6 13V8h12v5a4 4 0 0 1-4 4h-4a4 4 0 0 1-4-4Z" />
```

Comment:
Saving the world one byte at a time... 😆 
```suggestion
  <path d="M18 8v5a4 4 0 0 1-4 4h-4a4 4 0 0 1-4-4V8Z" />
```

Reply:
> Saving the world one byte at a time...

If only that was all it took…